core.js

```diff
@@ -16,6 +16,24 @@ const parse5 = require('parse5');
 const fetch = require('window-fetch');
 const {Request, Response, Headers, Blob} = fetch;
 
+const SITE_MODULES = path.join(__dirname, 'site_modules');
+
+const rmrf = path => {
+  try {
+    return require('rimraf').sync(path);
+  } catch(e) {
+    return e;
+  }
+}
+
+const mkdir = path => {
+  return require('mkdirp').sync(path);
+}
+
+// Nuke site_modules on startup.
+rmrf(SITE_MODULES)
```

Comment:
This should probably not be done synchronously.

Reply:
It should, because the alternative is to nest the entirety of core.js inside of a callback. :P

Reply:
But we are already bootin the site async, since it needs to go throught a request. There is no technical reason to not parallelize that.

Reply:
A more worrying reason is that this will rmrf the `site_modules` whenever a web worker loads. Suffice to say, yes, this needs more thought.